validation/static/styles.css

```diff
@@ -135,7 +135,23 @@ li.speaker_code {
   transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out;
 }
 
-.button-success:hover {
+.button-success-solid {
```

Comment:
A few notes on refactoring CSS:

I have found the philosophy of [BEM](http://getbem.com/naming/) and [Tailwind CSS](https://tailwindcss.com/) to be quite helpful in DRYing up my rules.

So I would make a `.button` class that contains all common code. And then I'd make classes called `.button--success` and `.button--fail` that would override the colours.

I'm not immediately sure if there's a distinction between `.button--solid` and... `.button--not-solid`, but either way, I assume _most_ of those style changes to be orthogonal to `.button--success`/`.button--fail` (e.g., there's little interaction).

Usage would be:

```
<button type="submit" class="button button--solid button--success">...</button>
<a href="#" class="button button--solid button--success">...</a>
```

etc.

Reply:
`button--solid` is to indicate the current state of a segment, so it's somewhere between "selected" and "unselected". That is, the background colour is just at 80% opacity.